helix-query.yaml

```diff
@@ -0,0 +1,23 @@
+version: 1
+
+indices:
+  allpages:
+    include:
+      - '/**'
+    exclude:
+      - /partials/**
+      - /drafts/**
+    target: /query-index.xlsx
+    properties:
```

Comment:
Could we also include the subtitle column? It targets `main h1 + h2`.

Reply:
It looks like the `description` that is being pulled into `query-index` is also being truncated. Could we avoid that?

Reply:
descriptions: I took a look at the descriptions on the author bios that end in "..." and it's using the correct value from the metadata. It looks like that value is automatically generated as a default when there is no `description` set in the metadata.